packages/editor/src/components/parts/task/useTaskData.ts

```diff
@@ -6,7 +6,7 @@ import { useConfigDataContext, useTaskDataContext } from '../../../context';
 import type { ResponsibleUpdater } from '../common/responsible/ResponsibleSelect';
 import type { PriorityUpdater } from './priority/PrioritySelect';
 
-type NotificationUpdater = DataUpdater<WfNotification>;
+export type NotificationUpdater = DataUpdater<WfNotification>;
```

Comment:
It seems as if you don't use this anywhere outside of this file, so I wouldn't export it here.